src/db/nedb.js

```diff
@@ -48,7 +49,15 @@ class DBHelper {
     }
   }
 
-  static async updateTopicByQuery(db, topic, query) {
+  static async updateObjectByQuery(db, query, update) {
+    try {
+      await db.update(query, { $set: update }, {});
+    } catch (err) {
+      logger.error(`Error update ${update} object by query:${query}: ${err.message}`);
```

Comment:
is it possible to get the db name from `db` object so we know which one failed on which table?